Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -1,15 +1,24 @@
-if (NOT Qt5_DIR AND MSVC)
+if (Qt6_DIR)
+	set(Qt_DIR "${Qt6_DIR}")
+endif()
+
+if (Qt5_DIR)
+	set(Qt_DIR "${Qt5_DIR}")
+endif()
+
+if (NOT Qt_DIR AND MSVC)
   if(_M_ARM_64)
-    set(Qt5_DIR "${CMAKE_SOURCE_DIR}/Externals/Qt/Qt5.15.0/msvc2019_arm64/lib/cmake/Qt5")
+    set(Qt_DIR "${CMAKE_SOURCE_DIR}/Externals/Qt/Qt5.15.0/msvc2019_arm64/lib/cmake/Qt5")
   else()
-    set(Qt5_DIR "${CMAKE_SOURCE_DIR}/Externals/Qt/Qt5.15.0/msvc2019_64/lib/cmake/Qt5")
+    set(Qt_DIR "${CMAKE_SOURCE_DIR}/Externals/Qt/Qt5.15.0/msvc2019_64/lib/cmake/Qt5")
```

Comment:
I'm guessing this stays on Qt5 for now, until the submodule is updated?

Reply:
Correct. From reading some release notes, I don't see any need to upgrade very soon unless something breaks in a new macOS release to be honest.